instrumentation-api/src/test/java/io/opentelemetry/instrumentation/api/caching/CacheTest.java

```diff
@@ -55,6 +65,9 @@ void bounded() {
     void unbounded() {
       Cache<String, String> cache = Cache.newBuilder().setWeakKeys().build();
 
+      assertThat(cache.computeIfAbsent("bear", unused -> "roar")).isEqualTo("roar");
+      cache.remove("bear");
+
       CaffeineCache<?, ?> caffeineCache = ((CaffeineCache<?, ?>) cache);
       String cat = new String("cat");
```

Comment:
I detect that this code is problematic. According to the [Performance (PERFORMANCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#performance-performance), [Dm: Method invokes inefficient new String(String) constructor (DM_STRING_CTOR)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#dm-method-invokes-inefficient-new-string-string-constructor-dm-string-ctor).
Using the java.lang.String(String) constructor wastes memory because the object so constructed will be functionally indistinguishable from the String passed as a parameter.  Just use the argument String directly.

Reply:
@lillieMaiBauer Can you please disable this organization from your code review bot? We have satisfactory checks in place, including exclusions that need to be made in some cases like this.

Reply:
ok, I will